core/capabilities/ccip/delegate.go

```diff
@@ -248,7 +258,7 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) (services
 		ragep2ptypes.PeerID(p2pID.PeerID()),
 		d.lggr,
 		hcr,
-		12*time.Second,
+		12*time.Second, // tickInterval matches ethereum block time
```

Comment:
We have a const for polling HomeChain in chainlink-ccip under `HomeChainPollingInterval`, might be worth considering as unification

Reply:
Used const with jitter like the doc suggests ;-)